execution_engine/src/core/runtime/externals.rs

```diff
@@ -968,6 +972,61 @@ where
                 self.record_era_info(era_id, era_info)?;
                 Ok(Some(RuntimeValue::I32(0)))
             }
+            FunctionIndex::NewDictionaryFuncIndex => {
+                let (output_size_ptr,): (u32,) = Args::parse(args)?;
```

Comment:
I appreciate args description comments in these methods, such as:
// args(0) = _description of what this arg is supposed to be_

Reply:
Addressed